metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/ObjectsImporter.java

```diff
@@ -305,7 +305,7 @@ public Sample importSample(String accession) {
 
     private Taxonomy extractTaxonomyFromSample(SampleType sampleType) {
         SampleType.SAMPLENAME sampleName = sampleType.getSAMPLENAME();
-        return new Taxonomy(sampleName.getTAXONID(), sampleName.getSCIENTIFICNAME());
+        return new Taxonomy(sampleName.getTAXONID(), sampleName.getSCIENTIFICNAME(), Taxonomy.Rank.SPECIES);
```

Comment:
The problem is that we can't be 100% sure that the `getTAXONID()` will return a **species** rank ID. It may also return a subspecies (I think you found an example of that, could you post it here?) or even a "no rank" lower level, especially in case of bacteria and other poorly classified organisms. For example, a submitter might want to specify that the data is from _Staphylococcus aureus 04-02981_ (a specific strain), which is a "no rank" taxon: https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=703339&lvl=3&lin=f&keep=1&srchmode=1&unlock

I suggest that we separate taxonomy information into two parts:
* Original taxonomy ID, stored as a plain integer
* Fixed-rank IDs for species, genus, etc., for indexing and searching

We can discuss this during standup tomorrow.

Reply:
ya I forgot to remove the rank over here, I could just return the taxonomyId for the later use.

Reply:
As far as I can see the taxonomy is still created with a fixed rank of "SPECIES", even though it can be something else